pallets/pallet-dip-consumer/src/origin.rs

```diff
@@ -16,7 +16,7 @@
 
 // If you feel like getting in touch with us, you can do so at info@botlabs.org
 
-use frame_support::{traits::EnsureOrigin, RuntimeDebug};
+use frame_support::{sp_runtime::RuntimeDebug, traits::EnsureOrigin};
```

Comment:
We can use `sp_runtime` directly.

Reply:
[d7dc75a](https://github.com/KILTprotocol/kilt-node/pull/627/commits/d7dc75ac91e2c01f80f30196df5401ea77fe9f2d)

Reply:
The pallet does not have sp_runtime. should I add it?

Reply:
Yes, since it's pulled in anyway.